recipes/influx_si/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "influx_si" %}
+{% set version = "5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ec782f72354b4a1f9f6a413865e66188738281a63ea082e6fb77042874c6f230"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: generic
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - scipy
+    - r-base
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo
+    - r-rmumps >=5.2.1_6
+    - r-arrapply
+    - r-limsolve
+    - r-multbxxc
+    - r-nnls
+    - r-slam
```

Comment:
You probably want these in `run:` rather than `host:`. The latter is only present during compilation and not on end-user systems.